test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
Please make sure that this file (test.md) is not present in your repository.

Reply:
Done 🥇